pootle/static/js/browser/components/TopContributors.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) Pootle contributors.
+ *
+ * This file is a part of the Pootle project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+
+import Avatar from 'components/Avatar';
+import { t } from 'utils/i18n';
+
+
+const TopContributors = React.createClass({
+
+  propTypes: {
+    items: React.PropTypes.array.isRequired,
+    allItemsLoaded: React.PropTypes.bool.isRequired,
+    onLoadMore: React.PropTypes.func,
+  },
+
+  getInitialState() {
+    return {
+      items: this.props.items,
+      allItemsLoaded: this.props.allItemsLoaded,
+    };
+  },
+
+  loadMore() {
+    const _this = this;
+    this.props.onLoadMore((items, allItemsLoaded) => {
+      _this.setState({items, allItemsLoaded});
+    });
+  },
+
+  render() {
+    function createRow(item, index) {
+      const title = `<span class="value untranslated">${item.suggested}</span> suggested<br/>
+              <span class="value translated">${item.translated}</span> translated<br/>
+              <span class="value fuzzy">${item.reviewed}</span> reviewed<br/>`;
+      return (
+        <tr key={`top-contibutor-${index}`}>
+          <td className="number">{ t('# %(position)s', { position: index + 1})}</td>
+          <td className="user top-scorer">
+            <Avatar
+              email={item.email}
+              label={item.display_name}
+              size={20}
+              username={item.username}
+            />
+          </td>
+          <td className="number">
+            <span title={title}>
+              { item.public_total_score > 0 ?
+                t("+%(score)s", {score: item.public_total_score}) :
+                item.public_total_score
+              }
+            </span>
+          </td>
+        </tr>
+      );
+    }
+
+    const { items } = this.props;
+    let loadMore;
+
+    if (!this.state.allItemsLoaded) {
+      loadMore = (
+        <div className="more-top-contributors">
+          <a onClick={this.loadMore}>
+            <span className="show-more">{ gettext("more...") }</span>
+          </a>
+        </div>
+      );
+    }
+    return (
+      <div>
+        <table className="top-scorers-table">
+          <tbody>
+            {this.state.items.map(createRow.bind(this))}
```

Comment:
There's no need to `bind(this)`. Also, having `createRow` as a method (even if it doesn't need access to `props` at the moment) will allow the reader to scan and understand the `render()` function easier.